ACMILabs/interactive-label
# #40 Fix linting, update README, update for new Label API shape

Resolves #40

Set up Interactive Labels project for local development.

### Acceptance Criteria
- [x] Make sure this project is setup for local development
- [x] Update/add testing and development instructions to README
- [x] Update `package.json` for security updates
- [x] Add background image making scripts to be exempt from linting

### Relevant design files
* None

### Testing instructions
Follow the README to see if you can:
1. Run a development server (Use `XOS_PLAYLIST_ID=44`)
1. Run Python linting and tests
1. Run Javascript linting and tests
1. Create background images using Harry's tools

### DoD
For requester to complete:
